Reply/app/src/main/java/com/example/reply/ui/navigation/ReplyNavigationComponents.kt

```diff
@@ -1,4 +1,20 @@
-package com.example.reply.ui
+/*
```

Comment:
Comment for the entire class: consider doing what we chatted about and moving all navigation information into the Destinations class (one object per each Destination with its icons, labels etc.) and creating a list of TopLevelDestinations, then iterating & calling .forEach { NavigationBarItem/ NavigationRailItem}. It would make this entire file 4x shorter and avoid a lot of repetition :). Also you'd have one class for centralizing all navigation related information in one place

Reply:
Thanks, updated it to use TopLevelDestinations.